### ISSUE TITLE
Adjust emoji replacement

### ISSUE THREAD

**Issue by Androkai** [DiscordSRV/DiscordSRV#1068]
Since version 1.16 Minecraft chat allows full UTF-8 range for characters. While the vanilla font still requires an update this allows adding non-"supplementary characters" emojis via [resource packs](https://github.com/Androkai/AndrosDiscordEmojis).
With `ParseEmojisToNames` option disabled DiscordSRV removes (parts? - see screenshot below) emojis which is fine for < 1.16 servers but may be annoying if you want to show at least these common emojis.

![image](https://user-images.githubusercontent.com/3952718/113906860-fee1bd80-97d4-11eb-9947-d46a923e8b62.png)
![image](https://user-images.githubusercontent.com/3952718/113906686-cb069800-97d4-11eb-8ce7-853c03cf38cf.png)

Therefore I suggest that we add another option or switch the current one to allow the following combinations:
- replace emojis with it's name/shortcode
- remove all emojis
- replace only custom emojis with it's name/shortcode and send the default ones as is

For testing configure in server.properties to add server resource pack with emoji font:
```properties
resource-pack=https\://github.com/Androkai/AndrosDiscordEmojis/releases/download/1.16.2%2B_v1/AndrosDiscordEmojis_mc1.16.2+_v1.zip
resource-pack-sha1=431E5753635D692FD84D0ED2E83B63784ABCB504
```

**Comment by Vankka** [comment]
It'd probably be best to have separate options for unicode emojis & custom emotes, that makes the most sense to me anyway (emojis would have :name:/hide/show, emotes would have :name:/hide). 

Also side though, there could be a way to replace a custom emote with something else entirely (eg. by replacing the emotes with a intermediary such as :id: and using the regex option) but would anybody even want that as a option?

**Comment by Androkai** [comment]
> Also side though, there could be a way to replace a custom emote with something else entirely (eg. by replacing the emotes with a intermediary such as 🆔 and using the regex option) but would anybody even want that as a option?

Sounds like a lot of manual effort nobody will do.

**Comment by Vankka** [comment]
Implemented in the above PR, available since v1.23.0
